src/org/labkey/test/tests/DomainDesignerTest.java

```diff
@@ -305,6 +305,13 @@ public void testBlankNameFieldOnAddedField() throws Exception
         assertTrue("expect error to contain [Please provide a name for each field.] but was[" + hasNoNameError + "]",
                 hasNoNameError.contains("Please provide a name for each field."));
 
+        String warningfieldMessage = noNameRow.setName("&totally not a sketchy sql injection attack")
```

Comment:
This text is misleading. We aren't testing anything injection related. Having weird characters in field names just makes scripts harder to write.

Reply:
new name: [&has weird characters that make scripts hard to write]